targets/unity-v2/source/Shared/Internal/Log.cs

```diff
@@ -1,7 +1,10 @@
+using System;
+
 namespace PlayFab.Internal
 {
     public static class Log
```

Comment:
Let's deprecate the whole class:
```suggestion
    [Obsolete("This logging utility has been deprecated. Use UnityEngine.Debug.Log")]
    public static class Log
```